frontend/src/metabase/css/core/arrow.module.css

```diff
@@ -1,37 +1,33 @@
 /* TODO: based on popover.css, combine them? */
 /* TODO: other arrow directions */
 
-.arrow-right {
+:global(.arrow-right) {
```

Comment:
there's no class exposed via css module. You referenced it like `CS.arrow_right`.

there are 2 things here.
 1. I think we should use camel case for CSS Module so this should be `.arrowRight`
 2. we need to add those classes in the CSS module (there are only global classes)